.rubocop.yml

```diff
@@ -30,3 +30,7 @@ Style/IndentationConsistency:
 
 Style/RedundantReturn:
   Enabled: true
+
+Style/StringLiterals:
+  Enabled: true
+  EnforcedStyle: single_quotes
```

Comment:
I know it's the most common style in Ruby style guides and it makes some sense.
I don't think it improves readability much.
But I am afraid it's going to be much more annoying than useful, by breaking the CI for such a detail (remember Findbugs warnings?).
I am not sure either how fast we'll all learn to always use the right style of quotes, but I can imagine it will take me a while to use `'` in `raise "error message"`.